### DISCUSSION TITLE
About properly erasing data

### DISCUSSION THREAD

**Discussion by lost-geographer** [Divested-Mobile/DivestOS-Build#69] [Q&A] [answered]
Hi ! Thank you very much for the DivestedOS project !

I would need some clarification. Assuming that the phone is encrypted: is resetting the phone to factory settings enough to erase the data properly or do I need to use (also?) Extirpater.

I've read that resetting an encrypted phone erases the encryption keys, making data recovery almost impossible, which would make the use of Extirpater optional (especially since its impact on storage lifespan might be significant) : is this correct ?

**Comment by SkewedZeppelin** [comment, marked as answer]
The use case for Extirpater is just to be double sure for a device you plan to
sell or give away.
Or if you know your device wasn't previously encrypted.

**Comment by SkewedZeppelin** [comment]
The use case for Extirpater is just to be double sure for a device you plan to
sell or give away.
Or if you know your device wasn't previously encrypted.
